### DISCUSSION TITLE
Rubocop looking in wrong directory for it's default configuration.

### DISCUSSION THREAD

**Discussion by patrick-motard** [LazyVim/LazyVim#4279] [Q&A]
Does anyone know where/how mason/neovim/rubocop determines what folder to look for the default rubocop config? Mason has installed rubocop 1.65.1, but I'm getting an error when rubocop runs in nvim stating that it can't find default.yml in ~/.local/share/nvim/mason/packages/rubocop/gems/rubocop-1.65.0/config/default.yml. It DOES exist in the 1.65.1 directory however.

**Comment by fidalgo** [comment]
I'm running into the same... I went nuts about this, even imploded my ruby installation and still for some reason mason rubocop is being picked up!

**Reply by fidalgo** [reply]
I've found out if we have rubocop running from one directory/project/etc where the versions are different, then when it tries to run, it will try to use that server version that is running.

@patrick-motard just check if rubocop is already running with `ps uax | rg rubocop`
